frontend/public/components/graphs/gauge.tsx

```diff
@@ -122,6 +128,7 @@ type GaugeChartProps = {
   }[];
   title?: string;
   usedLabel?: string;
+  totalUsedValue?: number;
```

Comment:
Sorry, I didn't realize we weren't passing the raw total and used to the component.

Would it be better to accept the `labels` function as an optional prop? That seems like a cleaner solution.

cc @TheRealJon who knows the chart code better.